core/src/components/select/select.ionic.scss

```diff
@@ -22,6 +22,17 @@
   --highlight-color-invalid: #{globals.$ion-semantics-danger-800};
 }
 
+// Select Hint Text
+// ----------------------------------------------------------------
+
+.select-bottom {
+  @include globals.typography(globals.$ion-body-sm-medium);
```

Comment:
Shouldn't this have a margin/padding top of --ion-space-100? Looking at the snapshots, the text feels too close to the select

Reply:
You're right! I was looking at Figma with dev mode disabled and didn't see it. 🙈 Added here: https://github.com/ionic-team/ionic-framework/pull/30280/commits/162c57b4be8d9c2854c080c7759b2f1c0a33d73c